ui/component/app/view.jsx

```diff
@@ -231,6 +224,7 @@ function App(props: Props) {
   useEffect(() => {
     if (!hasSignedIn && hasVerifiedEmail) {
       signIn();
+      analytics.setUser(userId);
```

Comment:
just move this inside of the `signIn` function. Right now the correct userId won't be included in this effect